app/gateway/2.7.x/configure/auth/kong-manager/oidc-mapping.md

```diff
@@ -0,0 +1,172 @@
+---
+title: OIDC Authenticated Group Mapping
+badge: enterprise
+---
+
+Using Kong's OpenID Connect plugin (OIDC), you can map identity provider (IdP)
+groups to Kong roles. Adding a user to Kong in this way gives them access to
+Kong based on their group in the IdP.
+
+Starting with {{site.base_gateway}} version 2.7, admin accounts are created automatically 
+when you map your identity provider (IdP) groups to Kong roles. You do 
+not need to create the users, groups, and roles separately. These users then accept invitations to join 
+Kong Manager and log in with their IdP credentials.
+
+If an admin's group changes in the IdP, their Kong admin account's associated
+role also changes in {{site.base_gateway}} the next time they log in to Kong
+Manager. The mapping removes the task of manually managing access in
+{{site.base_gateway}}, because it makes the IdP the system of record.
+
+Here's how OIDC authenticaticated group mapping works:
+
+1. Configure the OIDC plugin to connect with your IdP.
+1. When users log in to Kong Manager, they are assigned to the roles they belong to in your IdP.
+
+## Prerequisites
+
+* An IdP with an authorization server and users with groups assigned
+* [{{site.ee_product_name}} installed and configured](/gateway/{{page.kong_version}}/install-and-run)
+* Kong Manager enabled
+* RBAC enabled
+* (Kubernetes) [Helm](https://helm.sh/docs/intro/install/) installed
+
+## Apply OIDC Auth Mapping to Kong Gateway
+
+In the following examples, you specify the `admin_claim` and `authenticated_groups_claim` parameters 
+to identify which admin value and role name to map from the IdP to {{site.base_gateway}}.
+
+The `admin_claim` value specifies which IdP username value should map to Kong Manager. 
+Note that the username and password are required for the user to log into the IdP.
+
+The `authenticated_groups_claim` value specifies which IdP role should be assigned to the
+specified user. This value must be provided in the format provided in the examples.
```

Comment:
I just realized this description is incorrect. ` authenticated_groups_claim` value specifies which Idp claim the gateway should use for the role mapping.

so this might the reason that @lena-larionova confuses with the role mapping description. (https://github.com/Kong/docs.konghq.com/pull/3479#issuecomment-994086159)

so that `authenticated_groups_claim` can be specify like below snap shot (OKTA example)
![image](https://user-images.githubusercontent.com/52717686/146094902-21138c07-c799-4d35-9199-d06c6841778b.png)

the `{WORKSPACE_NAME}:{ROLE_NAME}` format is the requirement for the group/role names in the idp.

@Bradamant3 , @lena-larionova , Hope this resolve/answer ur question.